src/index.js

```diff
@@ -45,17 +45,18 @@
  * The main interface is the {@link analyze} function.
  * @module escope
  */
+"use strict";
 
 /*jslint bitwise:true */
 
-import assert from 'assert';
+const assert = require('assert');
 
-import ScopeManager from './scope-manager';
-import Referencer from './referencer';
-import Reference from './reference';
-import Variable from './variable';
-import Scope from './scope';
-import { version } from '../package.json';
+const ScopeManager = require('./scope-manager').default;
+const Referencer = require('./referencer').default;
+const Reference = require('./reference').default;
+const Variable = require('./variable').default;
+const Scope = require('./scope').default;
```

Comment:
Can you use imports like `require('./foo')` here instead of `require('./foo').default`? Then the modules could use `module.exports = Foo` instead of `module.exports.default = Foo`.